Cargo.toml

```diff
@@ -9,17 +9,13 @@ edition = "2018"
 crate-type = ["staticlib"]
 
 [features]
-default = ["liquid"]
-liquid = ["elements", "liquid-rpc"]
+liquid = ["elements"]
 android_logger = ["android_log"]
 stderr_logger = ["stderrlog"]
 
 [dependencies]
-bitcoin = "0.18"
-bitcoincore-rpc = "0.7.0"
-bitcoin_hashes = { version = "0.3", features = [ "serde" ] }
-secp256k1 = { version = "0.12", features = [ "rand" ] }
-jsonrpc = "0.11"
+bitcoin = { version = "0.20", features = [ "use-serde" ] }
+bitcoin_hashes = { version = "0.7", features = [ "serde" ] }
```

Comment:
You can remove this one. It's accessible as `use bitcoin::hashes::{...};` instead. So you make sure the versions can't get out of sync.

Reply:
good call, I think I remember having issues with this but I could try again...